src/components/video-home-card/video-home-card.component.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+
+import { VideosContext } from '../../providers/videos';
+import { setVideoToWatch } from '../../providers/videos/videos.actions';
+
+import {
+  VideoHomeCardContianer,
+  Thumbnail,
+  CardInformation,
+  Cardtext,
+  CardAvatar,
+} from './video-home-card.styles';
+
+function VideoHomeCard({ video }) {
+  const { videosDispatch } = useContext(VideosContext);
+  const {
+    videoId,
+    image,
+    title,
+    views,
+    timestamp,
+    channel: { title: channelTitle, image: channelImage },
+  } = video;
+
+  const handleMoveToWatchVideoOnClick = () => {
+    videosDispatch(setVideoToWatch(video));
+  };
+
+  return (
+    <VideoHomeCardContianer>
+      <Link to={`wv/${videoId}`} onClick={handleMoveToWatchVideoOnClick}>
+        <Thumbnail className='thumbnail' src={image} alt='' />
+      </Link>
+
+      <CardInformation>
+        <CardAvatar className='avatar' alt={channelTitle} src={channelImage} />
+        <Cardtext>
+          <Link to={`wv/${videoId}`} onClick={handleMoveToWatchVideoOnClick}>
+            <h4>{title}</h4>
```

Comment:
For some reason, the title of the Card contains some [HTML Entities](https://www.w3schools.com/html/html_entities.asp). Try to escape these characters.
<img width="287" alt="Screen Shot 2020-12-09 at 19 14 41" src="https://user-images.githubusercontent.com/6569533/101712554-02498000-3a53-11eb-8825-a72533aefa84.png">